src/include/cephd/libcephd.h

```diff
@@ -0,0 +1,48 @@
+#ifndef CEPH_LIBCEPHD_H
+#define CEPH_LIBCEPHD_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define LIBCEPHD_VER_MAJOR 0
+#define LIBCEPHD_VER_MINOR 1
+#define LIBCEPHD_VER_PATCH 0
+
+#define LIBCEPHFD_VERSION(maj, min, extra) ((maj << 16) + (min << 8) + extra)
+#define LIBCEPHFD_VERSION_CODE LIBCEPHD_VERSION(LIBCEPHD_VER_MAJOR, LIBCEPHD_VER_MINOR, LIBCEPHD_VER_PATCH)
+
+#if __GNUC__ >= 4
```

Comment:
similar case here with librados, librbd and libcephfs . they all have the `__GNUC__ >=4` check. I'm tempted to leave this the same also for consistency with the other libraries.